packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -10,17 +10,53 @@ import Modal, {
   ModalHeader
 } from 'cozy-ui/transpiled/react/Modal'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import { SubTitle } from 'cozy-ui/transpiled/react/Text'
 
 import accountMutations from '../connections/accounts'
 import triggersMutations from '../connections/triggers'
-import * as triggers from '../helpers/triggers'
+import * as triggersModel from '../helpers/triggers'
```

Comment:
A bit annoying that this model is not in cozy-doctypes, should we move it there?

Reply:
I agree with you. Let's do that in an other PR since we'll have comments / refacto to do on the copy pasted code ?